tests/foreman/cli/test_satellitesync.py

```diff
@@ -509,15 +532,15 @@ class TestContentViewSync:
 
     :CaseComponent: ContentViews
 
-    :Assignee: ltran
+    :Assignee: rmynar
```

Comment:
I'd say you can remove this assignee docstring because all test cases are technically under your ownership now and since you already have the assignee docstring at the beginning of the file, that itself should be sufficient.

Reply:
Right. I have removed the redundant docstrings and kept only the one at the beginning.